docs-site/content/28.0/api/search.md

```diff
@@ -208,7 +208,7 @@ When a `string[]` field is queried, the `highlights` structure will include the
+| q                    | yes      | The query text to search for in the collection.<br><br>Use `*` (wildcard operator) as the search string to return all documents. This is typically useful when used in conjunction with `filter_by`.<br><br>For example, to return all documents that match a filter, use:`q=*&filter_by=num_employees:10`.<br><br> Surround words with double quotes to do an exact phrase search. Eg: Setting `q` to `"tennis ball"` (including the surrounding double quotes) will only return documents that have those two words in that exact order, without any typo tolerance. <br><br>To exclude words in your query explicitly, prefix the word with the `-` operator, e.g. `q: 'electric car -tesla'`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
```

Comment:
Let's reword this to:

```
Use `q: "*"` (wildcard operator) as the search string...
```

That way it's clear that you can't use `*` somewhere in the middle of a keyword like in a regex.